src/rpm/builder.rs

```diff
@@ -260,9 +260,11 @@ impl RPMBuilder {
                 path: dest.clone(),
                 desc: "no parent directory found",
             })?;
+
         let (cpio_path, dir) = if dest.starts_with('.') {
```

Comment:
Using https://doc.rust-lang.org/std/primitive.str.html#method.strip_prefix could avoid the `unwrap` in line `267` entirely